### PR TITLE
Create is_hvac_sys_fluid_loop_attached_to_boiler(hvac_b.id).md

### PR DESCRIPTION
**Pull request by claperle**

### CODE REVIEW

File: docs/ruleset_functions/is_fluid_loop_attached_to_boiler(hvac_b.id).md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+**Inputs:**  
+- **B-RMR**: To evaluate if the fluid loop associated with the heating system associated with the HVAC system is attached to a boiler.   
+- **hvac_b.id**: The id of the hvac system to evaluate.  
+
+**Returns:**  
+- **is_fluid_loop_attached_to_boiler**: Returns TRUE if the fluid loop associated with the heating system associated with the HVAC system is attached to a boiler. Returns FALSE if this is not the case.   
+ 
+**Function Call:** None  
+
```
Comment (supriyagoel) [inline docs/ruleset_functions/is_fluid_loop_attached_to_boiler(hvac_b.id).md:13]: i believe this function might need an applicability criteria for checking if is fluid loop = true?
would it make sense to make self contained functions which check if HVAC system is connected to fluid loop and if the fluid loop is connected to boiler/PHW
then there wouldn't be a need for the applicability criteria

Comment (claperle) [reply docs/ruleset_functions/is_fluid_loop_attached_to_boiler(hvac_b.id).md:13]: @supriyagoel This applicability check is already done before this function is called in https://github.com/pnnl/ruleset-checking-tool/pull/480/files . If there is not a fluid loop then the other function would not call this one.

File: docs/ruleset_functions/is_fluid_loop_attached_to_boiler(hvac_b.id).md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+## Logic:   
+- For each boiler in B_RMR, save boiler to loop boiler dictionary: `for boiler_b in B_RMR.RulesetModelInstance.boilers: loop_boiler_dict[boiler_b.loop].append(boiler_b)`
+- Create an object associate with the heating_system fluid loop associated with hvac_b: `fluid_loop_b = hvac_b.heating_system[0].hot_water_loop`
+- Check if the fluid loop type is heating: `if fluid_loop_b.type = "HEATING":`
+    - Check if the fluid loop id is in the list created above, if yes then is_fluid_loop_attached_to_boiler equals TRUE  : `if fluid_loop_b.id in hot_water_loop_list_b: is_fluid_loop_attached_to_boiler = TRUE`
```
Comment (mkarpman) [inline docs/ruleset_functions/is_fluid_loop_attached_to_boiler(hvac_b.id).md:18]: I think here you mean to reference loop_boiler_dict - i.e., if any boiler in the dictionary references the fluid_loop_b.

**Review by mkarpman — APPROVED** [review]

File: docs/ruleset_functions/is_fluid_loop_attached_to_boiler(hvac_b.id).md (hunk 2 above)
Comment (claperle) [reply docs/ruleset_functions/is_fluid_loop_attached_to_boiler(hvac_b.id).md]: Ah yes, Updated. Should we be checking the hvac sys heating system or the terminal unit?

**Review by supriyagoel — APPROVED** [review]

File: docs/ruleset_functions/is_hvac_sys_fluid_loop_attached_to_boiler(hvac_b.id).md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+**Returns:**  
+- **is_hvac_sys_fluid_loop_attached_to_boiler**: Returns TRUE if the fluid loop associated with the heating system associated with the HVAC system is attached to a boiler. Returns FALSE if this is not the case.   
+ 
+**Function Call:** None  
+
+## Logic:   
+- Set is_hvac_sys_fluid_loop_attached_to_boiler equal to FALSE: `is_hvac_sys_fluid_loop_attached_to_boiler = FALSE`  
+- For each boiler in B_RMR, save boiler to loop boiler dictionary: `for boiler_b in B_RMR.RulesetModelInstance.boilers: loop_boiler_dict[boiler_b.loop].append(boiler_b)`
+- Create an object associate with the heating_system fluid loop associated with hvac_b: `fluid_loop_b = hvac_b.heating_system[0].hot_water_loop`
```
Comment (claperle) [inline docs/ruleset_functions/is_hvac_sys_fluid_loop_attached_to_boiler(hvac_b.id).md:17]: May need to be changed if we decide HW/CHW coils would be specified at the terminal level for Sys 1.

Comment (supriyagoel) [reply docs/ruleset_functions/is_hvac_sys_fluid_loop_attached_to_boiler(hvac_b.id).md:17]: as decided - systems with compressors would use HVAC system and hence the coils would specified at HVAC system level for system 1.

**Pull request merged**